public/rover.js

```diff
@@ -0,0 +1,58 @@
+const pathOfMarsRover = document.getElementById("path-of-mars-rover");
+const marsRoverSprite = document.getElementById("mars-rover-sprite");
+
+pathOfMarsRover.height = window.innerHeight;
+pathOfMarsRover.width = window.innerWidth;
```

Comment:
Unless I'm misunderstanding, does this way of modifying the image to fit the window make the image look stretched on certain screen shapes/sizes?

(I get the feeling you were doing this for practice and already know this, but...) I prefer to keep styling in the CSS rather than doing this in the JS (if possible!)

Reply:
Ah yes it does squish the image up on certain screen sizes - I'm thinking either to add a media query to set differently cropped images for different screen sizes, or to proportionally size the image - although this would mean the image won't fit the entire screen size. 

And yes, I decided to set this in the JS for practice, but will move it out to the CSS :)